src/scss/includes/utils.scss

```diff
@@ -54,6 +54,14 @@
   font-weight: normal;
 }
 
+.u-text--title {
```

Comment:
I know it's the design but I'm a bit worried this will be too big for the string to fit in some languages…

Reply:
let's see how it goes ?